README.md

```diff
@@ -1,3 +1,157 @@
+# WebARonTango
+
+**An experimental browser for Android that lets developers create Augmented Reality (AR) experiences using web technologies. An [ARCore Android version](https://github.com/google-ar/WebARonTango) and an [ARKit iOS version](https://github.com/google-ar/WebARonARKit) are also available.**
+
+<img alt="PointCloud" src="markdown/images/WebARExamplePicking.gif" style="float: left; object-fit: cover; width: 45%; height: 20em; margin-right: 1em; "><img alt="Occlusion" src="markdown/images/WebARExampleOcclusion.gif" style="width: 45%; height: 20em; object-fit: cover;">
+
+**Note:** This is not an official Google product. Nor is it a fully-featured web browser. Nor are the enabling JavaScript APIs standards, or on the standardization path. WebARonARTango is only meant to enable developer experimentation.
+
+## <a name="DifferencesBetweenWebARonTangoAndOthers">Differences between WebARonTango and WebARonARCore/WebARonARKit</a>
+
+All of the prototypes are build on top of the WebVR 1.1 spec and for the basic AR capabilities (motion tracking, camera feed rendering and hit testing) they all share a common workflow. However, WebARonTango exposes some additional functionalities and lacks others compared to the ARCore and ARKit based projects.
+
+### What does WebARonTango have that WebARonARCore/WebARonARKit does not?
+
+* Depth point cloud support: WebARonTango provides access to the depth point cloud that the infrared camera in the Tango spec includes.
+
+* **ADF support**: For now, WebARonTango exposes a very simple access to the Area Description capabilities of Tango. Maybe in the future, WebARonARCore may expose this functionality too.
```

Comment:
WebARonARCore AND WebARonARKit?

Reply:
For now it seems impossible as ARKit does not have ADF support :(. But I get your point, maybe not even mention it?